src/components/Audioplayer/RepeatDropdown/index.js

```diff
@@ -9,7 +10,37 @@ import { intlShape, injectIntl } from 'react-intl';
 import SwitchToggle from 'components/SwitchToggle';
 import LocaleFormattedMessage from 'components/LocaleFormattedMessage';
 
-const style = require('../style.scss');
+import { ControlButton } from '../index';
+
+const StyledPopover = styled(Popover)`
+  .popover-title {
+    font-family: ${props => props.theme.fonts.montserrat};
+    text-transform: uppercase;
+    color: ${props => props.color.theme};
+    padding-top: 15px;
+    padding-bottom: 15px;
+    font-size: 0.75em;
+  }
+  .popover-content {
+    a {
+      font-size: 0.8em;
+    }
+  }
+`;
+
+const Pill = styled(NavItem)`
+  a{
+    padding: 10px 15px;
+  }
+`;
+
+const disabled = css`
+opacity: 0.5;
```

Comment:
indent, if you use VS Code, it has support for styled-components

Reply:
I do. Hmm, not sure why didn't indent. Will fix